testcases/PowerNVDump.py

```diff
@@ -79,9 +79,10 @@ def setUp(self):
         self.pdbg = conf.args.pdbg
         self.basedir = conf.basedir
         self.c = self.cv_SYSTEM.console
-        #crash_content : a variable to point to right crash folder after core is
+        self.cv_SYSTEM.goto_state(OpSystemState.OS)
+        # crash_content : a variable to point to right crash folder after core is
         # generated by mpipl/kdump kernel. If dump failed this variable would be
-        #empty.
+        # empty.
```

Comment:
Mixing coding style fix and new feature. Its not so good to mix multiple fixes in same patch.

Reply:
:( again just before pushing I fixed few pep8 failures.. will revert them and send it as separate patch..